core/internal/leet/leftsidebarnav.go

```diff
@@ -33,6 +33,7 @@ func (s *LeftSidebar) updateSectionHeights() {
 	}
 
 	s.updateItemsPerPage()
+	s.normalizePagination()
```

Comment:
Why doesn't `updateItemsPerPage` do this? Preserving invariants shouldn't be the caller's responsibility.